GamepadGameInput.pde

```diff
@@ -0,0 +1,31 @@
+class GamepadGameInput extends GameInput {
+
+  ControlIO control;
+  ControlDevice gpad;
+
+  GamepadGameInput(String config, PApplet applet) {
```

Comment:
Doesn't look like you use `config` yet (and it doesn't work on my mac currently).